pkg/api/v1/offering.proto

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
I see no reason to put all the proto into a single one proto file, it will be a nightmare when we have many services and will have a super long proto file, I would prefer to put per service per proto file